scripts/emulated-performance.ps1

```diff
@@ -173,6 +174,63 @@ if ($UseTcp.Contains(1)) {
 # Wait for the server(s) to come up.
 Sleep -Seconds 1
 
+$OutputDir = Join-Path $RootDir "artifacts" "PerfDataResults" "$RemotePlatform" "$($RemoteArch)_$($Config)_$($RemoteTls)" "WAN"
+New-Item -Path $OutputDir -ItemType Directory -Force | Out-Null
+$OutputFile = Join-Path $OutputDir "WANPerf_$UniqueId.json"
```

Comment:
Can you just randomly generate a GUID instead of requiring a unique id to be passed in?